aries-site/src/layouts/navigation/NavPage.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Box, Text, Paragraph } from 'grommet';
+import { iconsMap } from '../../components/icons/iconsMap';
+
+import { navItemsData } from './navItemsData';
+
+const NavItem = ({ item, prefix }) => {
```

Comment:
Currently, it does not look like `NavItem` scales to mobile mocks. Can we create an issue to flag that this needs to be done? (e.g. Box row gap should be reduced, plus `<Text/>` size should be reduced.

Reply:
I incorporated this change into my recent commit so we don't need to create a separate issue for it.

Reply:
I incorporated this change into my recent commit so we don't need to create a separate issue for it.

Reply:
Great!